client/payment-details/summary/index.tsx

```diff
@@ -102,6 +104,10 @@ const PaymentDetailsSummary = ( {
 		: placeholderValues;
 	const renderStorePrice =
 		charge.currency && balance.currency !== charge.currency;
+	const { authorization } = useAuthorization(
+		charge.payment_intent || '',
+		charge.order?.number || 0
```

Comment:
Do you think there's a better alternative than passing the default values '' and 0?

Reply:
True, I don't like it neither. The problem is that the type specifies those properties as nullable...I'll have a look on Monday.

Reply:
Sure, thanks. It's a minor nitpick, so feel free to move on if it's not worth the time.

Reply:
I decided to use `as` to cast the values to the expected types here [8fe9dc5](https://github.com/Automattic/woocommerce-payments/pull/5006/commits/8fe9dc5b648af7da66ce7d7355a38ca22d53595d)

Reply:
Looks better :) Thanks for going the extra mile to fix this.